hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/utils/Pipelines.java

```diff
@@ -482,7 +485,7 @@ public static String opName(String operatorN, Configuration conf) {
   }
 
   public static String opUID(String operatorN, Configuration conf) {
-    return "uid_" + operatorN + "_" + getTablePath(conf);
+    return "uid_" + (++operatorIndex) + "_" + operatorN + "_" + getTablePath(conf);
   }
```

Comment:
this would break the compatbility, wondering whether we can check the naming duplication and only add distinct suffix for those.